sysbuild/suit.cmake

```diff
@@ -39,3 +39,50 @@ if(SB_CONFIG_SUIT_BUILD_RECOVERY)
     set_config_bool(recovery CONFIG_SUIT_LOCAL_ENVELOPE_GENERATE n)
   endif()
 endif()
+
+if(SB_CONFIG_SUIT_BUILD_AB_UPDATE)
+
+  set(SAMPLE_SOURCE_DIR ${APP_DIR})
+
+  set(APP_IMAGE ${DEFAULT_IMAGE}_slot_b)
+
+  string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
+  list(GET split_board_qualifiers 1 target_soc)
+  list(GET split_board_qualifiers 2 target_cpucluster)
+
+  set(BOARD_TARGET "${BOARD}/${target_soc}/${target_cpucluster}")
+
+  ExternalZephyrProject_Add(
+    APPLICATION ${APP_IMAGE}
+    SOURCE_DIR ${SAMPLE_SOURCE_DIR}
+    BOARD ${BOARD_TARGET}
+    BOARD_REVISION ${BOARD_REVISION}
+  )
+
+  set_config_string(${APP_IMAGE} CONFIG_SUIT_ENVELOPE_TARGET "application_b")
+  set_config_bool(${APP_IMAGE} CONFIG_NRF_REGTOOL_GENERATE_UICR n)
+
+  # Choose slot_b_partition as the active code partition
+  add_overlay_dts(${APP_IMAGE} "${SAMPLE_SOURCE_DIR}/sysbuild/ab_images_b.overlay")
```

Comment:
Missing files?
 - `sysbuild/ab_images_b.overlay`
 - `sysbuild/ab_images_b_net.overlay`

Reply:
Hmmm, both are already on the feat/suit_ab_images branch

Reply:
Probably has been added on the first initial commit - by mistake, of course. It should be part of this PR....